docs/src/rules/init-declarations.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: init-declarations
 rule_type: suggestion
+related_rules:
+- no-unused-vars
```

Comment:
```suggestion
- no-unassigned-vars
```

I'm guessing this was meant to be the new rule.

Reply:
Thanks for catching that 🤦